src/scripts/content.js

```diff
@@ -18,17 +18,57 @@ const createTeamsNotificationScript = () => {
   (document.head || document.documentElement).appendChild(s);
 };
 
-chrome.storage.sync.get(defaultOptions, optionsFromStorageOrDefaults => {
-  teamsExtension.options = optionsFromStorageOrDefaults;
-  console.debug('Teams extension options', teamsExtension.options);
-
-  createTeamsNotificationScript();
-});
+const createFocusLostHandler = windowId => () => {
+  console.debug('Focus lost: ' + windowId);
+  if (teamsExtension.options.switchToNewChatOnBlur) {
+    // This timeout is necessary to be able to focus a Teams window by clicking its notification. Otherwise it just minimizes right away.
+    setTimeout(() => {
+      if (!document.hasFocus()) {
+        // Don't switch to New Chat if there is a call is running
+        const hangupButton = top.document.querySelector('iframe.embedded-page-content').contentDocument.getElementById('hangup-button');
+        if (!hangupButton) {
+          const buttonNewChat = top.document.querySelector('[title="Nuova chat (Alt+N)"]');
```

Comment:
This selector is language-dependent so will not work in languages other than Italian. We can possibly use the selector `data-tid=lr-create-chat` if is the same also in the Italian version of teams.

Reply:
you are right, I committed the fix